sources/packages/backend/apps/db-migrations/src/sql/InstitutionLocations/Add-col-integration-contacts.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.institution_locations
+ADD
+    COLUMN integration_contacts VARCHAR(300) [];
+
+COMMENT ON COLUMN sims.institution_locations.integration_contacts IS 'Contacts who receive notification email on integration file processing.';
```

Comment:
are these contacts only used to send email notifications? I remember discussing this point in the grooming, but I don't remember exactly

Reply:
These contacts are exclusively for sending email notifications as per the story. 

@JasonCTang @andrewsignori-aot are we missing anything here?